homebrew_challenge/src/App.jsx

```diff
@@ -377,56 +377,124 @@ const App = () => {
+        "name": "Colin Lyman",
+        "linkedinRef": "https://www.linkedin.com/in/colin-lyman"
+      },
+      {
+        "name": "Jonathan Lau",
+        "linkedinRef": "https://www.linkedin.com/in/jonathancylau"
+      },
+      {
+        "name": "Craig Climie",
+        "linkedinRef": "https://www.linkedin.com/in/craig-climie"
+      }
+    ];
+
+    const sortedTeamMembers = teamMembers.sort((a, b) => a.name.localeCompare(b.name)).map((data, index) => {
+      return(
+        <li key={index}>
           <a
-            href={linkedInRef}
+            href={data.linkedinRef}
             target="_blank"
             rel="noopener noreferrer"
-            className="link "
+            className="link"
           >
-            {name}
+            {data.name}
           </a>
         </li>
-      );
-    }
+      )
+    });
 
     return (
-      <div fluid className="about-us">
-        <p>
-          The COVID-19 Dashboard for Scotland was created by the following STA
-          volunteers:
-        </p>
-        <ul>
-          {person(
-            "Rhi Batstone",
-            "https://www.linkedin.com/in/rhiannon-batstone-076191120"
-          )}
-          {person("Ric Clark", "https://www.linkedin.com/in/richard--clark")}
-          {person("Craig Climie", "https://www.linkedin.com/in/craig-climie")}
-          {person("Jonathan Lau", "https://www.linkedin.com/in/jonathancylau")}
-          {person("Colin Lyman", "https://www.linkedin.com/in/colin-lyman")}
-          {person(
-            "Cristina Perez",
-            "https://www.linkedin.com/in/cristina-perez-11229846"
-          )}
-          {person(
-            "Luke Pritchard-Woollett",
-            "https://www.linkedin.com/in/lukepritchardwoollett"
-          )}
-          {person(
-            "Euan Robertson",
-            "https://www.linkedin.com/in/euan-robertson-5845582"
-          )}
-          {person(
-            "Gabriela Satrovskaja",
-            "https://www.linkedin.com/in/gabriela-satrovskaja"
-          )}
-          {person("Allan Stevenson", "https://www.linkedin.com/in/alstev")}
-          {person("Donal Stewart", "https://www.linkedin.com/in/donalstewart")}
-          {person("Becky Still", "https://www.linkedin.com/in/rebeccastill1")}
-        </ul>
+      <div fluid="true" className="about-us">
+        <hr/>
+        <h1>About us</h1>
+        <p>This dashboard has been developed by members of the Scottish Tech Army to improve awareness of the impacts of Covid-19.</p>
+        <h2>The Scottish Tech Army</h2>
+        <p>Founded by Edinburgh based entrepreneurs, Alistair Forbes and Peter Jaco, the Scottish Tech Army Limited is a not for profit company that is building a volunteer Covid-19 technical response team that will work to help the Scottish Government, Local Authorities and other organisations across the country with rapid technical development projects to address current Covid-19 related challenges and post pandemic economic recovery. </p>
```

Comment:
tiny point - I think local authorities should be in lowercase

Reply:
Good point, thank you!